application/run.sh

```diff
@@ -36,7 +24,7 @@ if [[ $APP_ENV == "dev" ]]; then
     apt-get -y -q install php-xdebug
 fi
 
-apache2ctl start
+apache2ctl -k start -D FOREGROUND
```

Comment:
Does this remove the need for the `while true` below? (I have no idea, just asking.)

Reply:
I don't think so, but I think I'll try to confirm.

Reply:
I confirmed that it is still helpful to have the loop. I believe it's so the script doesn't exit and can continue listening for the kill signal.